src/pages/status.js

```diff
@@ -0,0 +1,180 @@
+import componentMetadata from '@primer/component-metadata'
+import {Box, Heading, Text, themeGet} from '@primer/components'
+import StatusLabel from '@primer/gatsby-theme-doctocat/src/components/status-label'
+import fetch from 'isomorphic-unfetch'
+import React from 'react'
+import styled from 'styled-components'
+import Layout from '../components/Layout'
+import LinkLight from '../components/LinkLight'
+
+// TODO: Make table header sticky
+const Table = styled.table`
+  display: block;
+  width: 100%;
+  overflow: auto;
+  position: relative;
+  border-collapse: collapse;
+
+  th {
+    font-family: ${themeGet('fonts.mono')};
+    font-weight: ${themeGet('fontWeights.normal')};
+    color: ${themeGet('colors.blue.3')};
+  }
+
+  th,
+  td {
+    padding: ${themeGet('space.2')} ${themeGet('space.3')};
+  }
+
+  th:first-child,
+  td:first-child {
+    padding-left: 0;
+  }
+
+  th:last-child,
+  td:last-child {
+    padding-right: 0;
+  }
+
+  td {
+    border-top: 1px solid ${themeGet('colors.gray.7')};
+    vertical-align: top;
+  }
+
+  th {
+    position: sticky;
+    top: 0;
+  }
+
+  img {
+    background-color: transparent;
+  }
+`
+
+export default function StatusPage() {
+  const [components, setComponents] = React.useState(null)
+
+  React.useEffect(() => {
+    getComponents().then((components) => setComponents(components))
+  }, [])
+
+  return (
+    <Layout
+      pageContext={{
+        frontmatter: {title: 'Component status', description: 'Status of components in the Primer Design System'},
+      }}
+    >
+      <Box className="container-xl" px={5} pb={8}>
+        <Box pt={8} pb={6}>
+          <Heading fontSize={[48, 56]} color="blue.4" lineHeight={1} mb={3}>
+            Component status
+          </Heading>
+          <Text as="p" fontSize={3} color="blue.2">
+            Status of components in the Primer Design System.
+            <br />
+            Check out the <LinkLight href="#">component lifecycle</LinkLight> for more information about each status.
+          </Text>
+        </Box>
+        {components ? (
+          <Table>
+            <colgroup>
+              <col style={{width: '15%'}} />
+              <col style={{width: '15%'}} />
+              <col style={{width: '15%'}} />
+              <col style={{width: '55%'}} />
+            </colgroup>
+            <thead>
+              <tr>
+                <th align="left">Component</th>
+                {/* TODO: How would we add a Figma column? Where would that data come from ? */}
+                <th>ViewComponent</th>
+                <th>React</th>
```

Comment:
Could we have Figma too?

Reply:
[There's a slot in the component-metadata schema for Figma](https://github.com/primer/component-metadata) but it's just the URL and not the status; we could change the schema to add that in there? Or is there a different way that we could magically pull the data for straight from Figma? 🤔 

I don't think it should be a blocker for this PR though because when we update/add to the component-metadata package we'll have to bump the version in this repo, and we can make any additional changes then.

Reply:
Adding status to Figma in the scheme sounds good to me, but I'd like us to wait a few hours to see what @ashygee suggests. I think it would be great with ship with Figma as we treat is an implementation too and it's useful for our designer audience, if it doesn't involve a lot of work.

Reply:
Where would be a good place for the Figma status labels to link to?

Reply:
For Figma status I think that would be a little difficult. I have made a consideration for adding the status to a component in the new layout I've started to use in Primer Web but I've been relating it more to the status of the component in code, which is tricky now that we have a divergence of status between PRC and PVC. 

![Component status label in Figma next to component example](https://user-images.githubusercontent.com/10384315/146433965-f8c71659-500e-4c80-b049-03d16bbf8df5.png)

Another thing to note is that we don't have actual status criteria for Figma. I think on this table, instead of a status, it would be better to state, possibly with the use of simple ✅  and ❌  emojis if the component exists in Figma. Maybe in the metadata there could be a key that passes a boolean for `inFigma: true/false` and then we can link to the page in Figma. I'd love to see more tooling to automate this so there would be less manual work but for now I think just having stock of the inventory would be more valuable in Figma than it's status since, ideally, the Figma component would be the most complete compared to code. (Although this is not the case everywhere, I'm looking at you buttons.)